azure-pipelines.yml

```diff
@@ -263,15 +263,15 @@ stages:
                 -azdoProject "internal"
                 -barToken $(MaestroAccessToken)
                 -githubPAT $(BotAccount-dotnet-bot-repo-PAT)
-    - stage: Promote_Arcade_To_Latest
-      displayName: Promote Arcade to '.NET Eng - Latest' channel
+    - stage: Push_to_net6_eng_channel
+      displayName: Promote Arcade to '.NET 6 Eng' channel
```

Comment:
This looks right, but there seems to be a mismatch between "promote" and "push"?

Reply:
I followed what we have for release/5.0 branch to keep that consistent..  Changed it back to promote